src/full_client.toit

```diff
@@ -416,7 +416,11 @@ class Session_:
     persistence_store_.remove_persisted_with_id id
 
   next_packet_id -> int:
-    return next_packet_id_++
+    while true:
+      candidate := next_packet_id_
+      next_packet_id_ = (next_packet_id_ + 1) & 0xFFFF
```

Comment:
Is the id always sent as a 16 bit value? Maybe use a named constant for the mask.

Reply:
done.